packages/rainbowkit/src/components/ConnectOptions/ConnectDetails.tsx

```diff
@@ -606,8 +620,10 @@ export function DownloadOptionsDetail({
       >
         {extensionDownloadUrl && (
           <DownloadOptionsBox
-            actionLabel={`Add to ${browser}`}
-            description="Access your wallet right from your favorite web browser."
+            actionLabel={`${i18n.t(
+              'intro.wallet.get_started.add_to',
+            )} ${browser}`}
```

Comment:
String replacement should look like this:
```ts
"Add to %{browser}"
i18n.t("intro.wallet.get_started.add_to", { browser });
```
We'll want to correct any instances of this

Reply:
Ah yeah missed that one 👍